UIScreenExtension.podspec

```diff
@@ -13,5 +13,5 @@ Pod::Spec.new do |s|
   s.author       = { "Jens Schwarzer" => "jens.schwarzer@marchv.com" }
   s.platform     = :ios, "9.0"
   s.source       = { :git => "https://github.com/marchv/UIScreenExtension.git", :tag => s.version }
-  s.source_files = "UIScreenExtension", "UIScreenExtension/**/*.{h,swift}"
+  s.source_files = "UIScreenExtension", "UIScreenExtension/**/*.{h,swift}", "Sources/**/*.{h,swift}"
```

Comment:
Is this change needed? :)

Reply:
Yes, I think so. I moved the file in the root directory:
from: `UIScreenExtension/UIScreenExtension.swift`
to: `Sources/UIScreenExtension/UIScreenExtension.swift` to conform SPM file structure.

So Cocoapods can find the extension file itself (the pod will include all `.h` and `.swift` files in `Sources` folder and subfolders if at some point more files will be added to the project).